src/com/google/enterprise/gsafeed/groups/Principal.java

```diff
@@ -143,8 +151,10 @@ public String getvalue() {
    * Sets the value of the value property.
    *
    * @param value allowed object is {@link String}
+   * @return this object
    */
-  public void setvalue(String value) {
+  public Principal setvalue(String value) {
```

Comment:
Is this some strange name-mapping issue with JAXB, so that it's not setValue?

Reply:
Yes.